### ISSUE TITLE
Controller configuration issues

### ISSUE THREAD

**Issue by SecularSteve** [mmatyas/pegasus-frontend#762]
Hi, and thanks for reporting errors! You can help us locating the bug even more by providing detailed information in the sections below:

### The issue

![controller issue](https://user-images.githubusercontent.com/33793273/102632269-2857da00-414f-11eb-91b4-65ccb4a4f72b.png)

I tried to re-configure my PS3 SIXAXIS controller in the "Gamepad Layout" menu now that it's properly working in Pegasus. Except that when I tried to push down the Right Trigger and the B (Circle) Button, now they remain active until I press and hold down the buttons the entire time. They fix themselves at pure random, and my controller doesn't have any hardware issues.

Does this happen on any other controllers, or is this a specific case?

### System info

Windows 10 build 2004

### Pegasus version

Alpha 15 Weekly Build #51 (Stable)

### Pegasus log

You can find a log file with detailed information about the launch of Pegasus under the following locations:

- Windows: `C:\Users\[username]\AppData\Local\pegasus-frontend\lastrun.log`
- Linux: `~/.config/pegasus-frontend/lastrun.log`
- macOS: `~/Library/Preferences/pegasus-frontend/lastrun.log`
- Android: `<storage>/pegasus-frontend/lastrun.log`
- portable mode: `<directory of the program>/config/lastrun.log`

If possible, please attach the file to this bug report.

[lastrun.log](https://github.com/mmatyas/pegasus-frontend/files/5716574/lastrun.log)

**Comment by mmatyas** [comment]
Yes, the green highlight likes to be stuck on screen in some cases, but other than that it shouldn't cause usability issues.

**Comment by SecularSteve** [comment]
But it does. The buttons are just stuck "pressed" that way until I keep pressing and re-configuring them to jam them out of their "green" 

I just found out that a good workaround is using "Enter" instead of A to configure the buttons individually (and hitting it just a millisecond after hitting enter). A nice option to at least partially solve this bug should be a "Hold Y to reset to Default Configuration" and a "Hold Down X to re-configure all of the buttons one by one!" with a simple 5 second time-out like in RetroArch and ES.

Speaking of which, they're handing out RetroArch beta keys on Steam again, I snagged one, and it works better than I thought (even in Pegasus).

![Screenshot 2020-12-20 165646](https://user-images.githubusercontent.com/33793273/102717732-6d614500-42e4-11eb-8541-21881b982b59.png)

Are there any plans to integrate Metadata directly from the Steam Version of RetroArch?

**Comment by mmatyas** [comment]
I've recently updated the gamepad handling module in the latest version, could you see if it changed anything for you?

**Comment by SecularSteve** [comment]
Just tried the Continous Build, and not really.

I mean, I solved it on my own a while ago, but wouldn't it be just easier to put in a "Hold Y to Configure/Calibrate Controller" or something?

With a "Hold for 3 Seconds" thing?

**Comment by HVR88** [comment]
This issue also happens with XBOX Series X|S controllers and the right and left triggers.

**Comment by mmatyas** [comment]
The gamepad screen got a redesign in the latest version. Do you run into any issues with that too?

**Comment by mmatyas** [comment]
I'll assume this got fixed, but feel free to report if there are still issues.
